addons/account/views/account_payment_method.xml

```diff
@@ -12,4 +12,26 @@
         </field>
     </record>
 
+    <record id="view_account_payment_method_line_kanban" model="ir.ui.view">
+        <field name="name">account.payment.method.line.kanban</field>
+        <field name="model">account.payment.method.line</field>
+        <field name="arch" type="xml">
+            <kanban>
+                <field name="name"/>
+                <field name="journal_id"/>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div t-attf-class="oe_kanban_content oe_kanban_global_click">
+                            <div class="row">
+                                <div class="col-6">
```

Comment:
Someone from bug fix told me to apply this template, it is used in several kanban mobile views, so I did not think about it too much. (For example:  https://github.com/odoo/odoo/blob/53f5e5aaba4edec04722c1403bff01e3b36a2f5a/addons/payment/views/payment_transaction_views.xml#L96 )
But in fact your modifications are correct, I will change it, it will make it clearer as the col-6/rows are not needed here.